ql/src/java/org/apache/hadoop/hive/ql/parse/BaseSemanticAnalyzer.java

```diff
@@ -2092,4 +2110,79 @@ protected String getFullTableNameForSQL(ASTNode n) throws SemanticException {
     }
   }
 
+  public QB getQB(){
+    return null;
```

Comment:
should it be abstract?

Reply:
wait, this is not even needed anymore (it was only needed when I made the recursive QB traversal in the BaseSemanticAnalyzer), removing